pkg/controller/hibernation/hibernation_controller_test.go

```diff
@@ -461,10 +740,18 @@ func TestReconcile(t *testing.T) {
 				},
 				csrUtil: mockCSRHelper,
 			}
-			_, err := reconciler.Reconcile(context.TODO(), reconcile.Request{
+			result, err := reconciler.Reconcile(context.TODO(), reconcile.Request{
 				NamespacedName: types.NamespacedName{Namespace: namespace, Name: cdName},
 			})
 
+			// Need to do fuzzy requeue after matching
+			if test.expectRequeueAfter == 0 {
+				assert.Zero(t, result.RequeueAfter)
+			} else {
+				assert.GreaterOrEqual(t, result.RequeueAfter.Seconds(), (test.expectRequeueAfter - 10*time.Second).Seconds(), "requeue after too small")
+				assert.LessOrEqual(t, result.RequeueAfter.Seconds(), (test.expectRequeueAfter + 10*time.Second).Seconds(), "request after too large")
```

Comment:
There's an `assert.InDelta` that lets you validate a number to within a specified level of precision. [Example](https://github.com/openshift/hive/blob/c0c34366183fea87eb7846a14972bdb6b8999a32/pkg/controller/argocdregister/argocdregister_controller_test.go#L251).